Cargo.toml

```diff
@@ -1,28 +1,28 @@
 [package]
-name = "etcd-rs"
-version = "0.3.0"
-authors = ["ccc <currantxx@gmail.com>"]
+name = "etcd-client"
+version = "0.1.0"
+authors = ["Qi Tian <mrwwheat@gmail.com>"]
 edition = "2018"
 keywords = ["etcd", "future", "async"]
-repository = "https://github.com/luncj/etcd-rs"
-homepage = "https://github.com/luncj/etcd-rs"
-description = "etcd client for rust"
-documentation = "https://docs.rs/etcd-rs"
+repository = "https://github.com/datenlord/etcd-client"
+homepage = "https://github.com/datenlord/etcd-client"
+description = "etcd client for datenlord"
 license = "MIT"
+categories = ["filesystem"]
 
 [dependencies]
-tonic = { version = "0.3", features = ["tls"] }
-bytes = "0.5"
-prost = "0.6"
-tokio = { version = "0.2", features = ["stream"] }
+anyhow = "1.0.31"
+async-compat = "0.1.4"
 async-stream = "0.2"
 async-trait = "0.1"
-futures = "0.3"
-thiserror = "1.0"
+bytes = "0.5"
+futures = "0.3.5"
+grpcio = "0.6.0"
 http = "0.2"
-
-[dev-dependencies]
-tokio = { version = "0.2", features = ["full"] }
+protobuf = "2.16.2"
+smol = "1.2.4"
+thiserror = "1.0"
```

Comment:
Why `thiserror` in dev-dependencies?

Reply:
thiserror is in dependencies, this repo does not have dev-dependencies at all... Where did you see the dev-dependencies?

Reply:
my bad